app/src/main/res/layout/recent_bookmarks_header.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This can be in a different PR. There are a number of things that my components are going to add that are missing from first glance in the "Jump back in" pr and I'd rather get this landed first then deal with that.

Reply:
In that case, I think you can duplicate https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/res/layout/recent_tabs_header.xml more or less here. It will help you cut back on the number of variables and really simplify your layout.

Reply:
I'd prefer to keep it as-is